aerogear/mobile-security-service
# feat(config): Added custom config using environment variables (AEROGEAR-8777)

## Motivation

https://issues.jboss.org/browse/AEROGEAR-8777

## What

Made the UI configurable using environment variables.

## Why

Certain values were hard-coded (such as the API URL). These should be easily configurable.

## Verification Steps
Add the steps required to check this change. Following an example.
 
Add a `.env` file to the root of the UI project:

```sh
cp ui/.env.example ui/.env
```

2. Modify the value for `REACT_APP_NAME` to be "test"
3. Run the application using `npm start`.
4. Has the header title updated to say "test"

## Checklist:


## Progress
